apko/private/apko_image.bzl

```diff
@@ -39,6 +40,8 @@ def _impl(ctx):
     keyrings = ctx.attr.contents[OutputGroupInfo].keyrings
 
     inputs = [ctx.file.config]
+    if ApkoConfigInfo in ctx.attr.config:
+        inputs += ctx.attr.config[ApkoConfigInfo].files.to_list()
```

Comment:
please collect depsets and avoid resolution to lists (as long as possible)

Reply:
Done in apko_run, but here I think it is as long as possible already. There is a single depset here that we pass to action's input, so we must flatten it to list IIUC.